guide/voice/the-basics.md

```diff
@@ -24,10 +24,10 @@ client.on('message', async message => {
 });
 ```
 
-If you try to join a channel the bot is already in, nothing will happen, so we don't need to run this method repeatedly.
+If you try to join a channel the bot is already in, nothing will happen, so you don't need to run this method repeatedly.
 
 ::: tip
-In the above example, we accessed a `VoiceState` through `member.voice`. A voice state exists for members connected to a voice channel and tells you which channel they're connected to, if the member is speaking, and whether they're deafened or muted. Consult the documentation for a full description of what voice states can do!
+In the above example, a `VoiceState` is accessed through `member.voice`. A voice state exists for members connected to a voice channel and tells you which channel they're connected to, if the member is speaking, and whether they're deafened or muted. Consult the documentation for a full description of what voice states can do!
```

Comment:
```suggestion
In the above example, a `VoiceState` is accessed through `member.voice`. Voice states include information about the channel the member is connected to (if any), if the member is speaking and whether they're deafened or muted. Consult  <docs-link path="class/VoiceState">the documentation</docs-link> for a full description of what voice states can do!
```
- factually incorrect; voice states exist even if the member is not connected
- add docs link